File: Sample/ShareDataSample/app/src/main/java/m/k/android/sample/sharedatasample/MainActivity.java
package m.k.android.sample.sharedatasample;

import android.content.Context;
import android.content.SharedPreferences;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;

import butterknife.Bind;
import butterknife.ButterKnife;
import butterknife.OnClick;

public class MainActivity extends AppCompatActivity {

    public static final String BROADCAST_TYPE = "via_broadcast_preference";

    @Bind(R.id.edit)
    EditText mEdit;
    @Bind(R.id.load_btn)
    Button mLoadBtn;
    @Bind(R.id.save_btn)
    Button mSaveBtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ButterKnife.bind(this);
    }

    @OnClick(R.id.load_btn)
    public void load() {
        mEdit.setText(pref(this, BROADCAST_TYPE).getString("sample", "NO DATA."));
    }

    @OnClick(R.id.save_btn)
    public void save() {
        pref(this, BROADCAST_TYPE).edit().putString("sample", mEdit.getText().toString()).commit();
    }

    private SharedPreferences pref(Context context, String name) {
        return context.getSharedPreferences(name, MODE_WORLD_READABLE|MODE_MULTI_PROCESS);
    }
}

File: LibrarySample/KoinSample/hello/src/main/java/com/github/mag0716/koinsample/MainActivity.kt
package com.github.mag0716.koinsample

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import org.koin.android.ext.android.inject

class MainActivity : AppCompatActivity() {

    companion object {
        private const val TAG = "Hello"
    }

    val helloService: HelloService by inject()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Log.d(TAG, "onCreate : ${helloService.hello()}($helloService)")
    }
}
